SomyaSinghal/Day1/01.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+
+int main() {
+    char name[50];
+    printf("Hello! What's your name? ");
+    // Your code to get the name input here.
+    scanf("%[^\n]s\n", &name);
+    // Write your answer to question as comment.
+
+    printf("Welcome, %s!\n", name);
+
+    // Your code for the rest of the questions and their answers go here.
+    char ans2[100];
+    printf("Why do you want to learn C? ");
+    scanf("%[^\n]s\n", &ans2);
```

Comment:
Same problem here, the \n you gave will come as input to this scanf and [^\n] will take that as to something to ignore and then next \n will go to third scanf as input.